addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -259,6 +259,13 @@ class PaletteSelectionScreen extends Component {
         if (logoSelectInput.files.length === 1) {
             const previousLogoAttachmentId = this.state.logoAttachmentId;
             const file = logoSelectInput.files[0];
+            if (file.size > 2500000) {
+                this.notification.add(_t("The logo is too large. Please upload a logo smaller than 2.5 MB."), {
+                    title: file.name,
+                    type: "warning",
+                });
+                return;
+            }
```

Comment:
```suggestion
            if (file.size > 2500000) {
                this.notification.add(
                    _t("The logo is too large. Please upload a logo smaller than 2.5 MB."),
                    {
                        title: file.name,
                        type: "warning",
                    }
                );
                return;
            }
```

And maybe add a `const MAX_LOGO_SIZE` ? I let you decide.